### PR TITLE
EDL HA Doc

### PR DESCRIPTION
**Pull request by tkatzir**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-24767

### CODE REVIEW

File: Packs/EDL/Integrations/EDL/EDL_description.md [hunk 1]
```diff
@@ -47,6 +47,15 @@ To access the Generic Export Indicators Service by instance name, make sure *Ins
 
 </~XSOAR>
 
+#### Note:
+When using more than one server in HA mode, the EDL should be configured to listen on a route, not on a port.
+
+In "route" listen mode, a request received by any of the app-servers will be redirected to the one currently running the nginx container.
```
Comment (ShirleyDenkberg) [inline Packs/EDL/Integrations/EDL/EDL_description.md:53]:
```suggestion
In "route" listen mode, a request received by any of the app servers will be redirected to the one currently running the NGINX container.
```

File: Packs/EDL/Integrations/EDL/EDL_description.md [hunk 2]
```diff
@@ -47,6 +47,15 @@ To access the Generic Export Indicators Service by instance name, make sure *Ins
...
+This does not happen in "port" listen mode, that's why "route" mode should be used.
+
+If the app-server running the container fails, the container should restart on a different app-server.
```
Comment (ShirleyDenkberg) [inline Packs/EDL/Integrations/EDL/EDL_description.md:56]:
```suggestion
If the app server running the container fails, the container should restart on a different app server.
```

File: Packs/EDL/Integrations/EDL/EDL_description.md [hunk 3]
```diff
@@ -47,6 +47,15 @@ To access the Generic Export Indicators Service by instance name, make sure *Ins
 
 </~XSOAR>
 
+#### Note:
+When using more than one server in HA mode, the EDL should be configured to listen on a route, not on a port.
```
Comment (ShirleyDenkberg) [inline Packs/EDL/Integrations/EDL/EDL_description.md:51]:
```suggestion
When using more than one server in High Availability (HA) mode, the External Dynamic List (EDL) should be configured to listen on a route, not on a port.
```

File: Packs/EDL/ReleaseNotes/3_2_7.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Generic Export Indicators Service
+
+- Documentation and metadata improvements.
```
Comment (ShirleyDenkberg) [inline Packs/EDL/ReleaseNotes/3_2_7.md:5]:
```suggestion
Documentation and metadata improvements.
```

**Comment by ShirleyDenkberg** [comment]
@DeanArbel Doc review completed.

**Review by DeanArbel — APPROVED** [review]

**Pull request merged**